app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -675,6 +683,13 @@ const CallScreen = ({
                 {!isLandscape && currentCall.reactionStream.length > 0 &&
                     <EmojiList reactionStream={currentCall.reactionStream}/>
                 }
+                {incomingCalls.incomingCalls.map((ic) => (
+                    <CallNotification
+                        key={ic.callID}
+                        incomingCall={ic}
+                        onCallsScreen={true}
+                    />
+                ))}
```

Comment:
the call screen seems that it can also be filled with notifications of other calls

Reply:
Yes, as designed. But like the channels screen, the user is supposed to be bothered by them, and either click on them or dismiss them.